bgruening.yaml

```diff
@@ -232,3 +232,6 @@ tools:
     owner: bgruening
     tool_panel_section_label: Single-cell
 
+ - name: hypo
+   owner: bgruening
```

Comment:
and this should be `iuc`

Reply:
yes, that makes sense. I just fixed it. Thanks